src/components/seances/DateSelector.vue

```diff
@@ -0,0 +1,161 @@
+<script setup lang="ts">
+import { computed, ref, watch } from "vue";
+import BaseButton from "../base/BaseButton.vue";
+import BaseSelect from "../base/BaseSelect.vue";
+import Datepicker from "@vuepic/vue-datepicker";
+import "@vuepic/vue-datepicker/dist/main.css";
+import CalendarIcon from "@/assets/icons/calendar.svg?component";
+import { useSeancesStore } from "@/stores/seances";
+import { useMovieStore } from "@/stores/movies";
+import type { DaysList } from "@/types/days-list";
+const seancesStore = useSeancesStore();
+const movieStore = useMovieStore();
+
+const todayDate = new Date();
+
+const daysList = computed(() => {
+  const daysList = [];
+  daysList.push({
+    id: 1,
+    date: todayDate.toISOString().substring(0, 10),
+    name: "Today",
+  } as DaysList);
+  for (let i = 1; i < 6; i++) {
+    const date = new Date(todayDate.setDate(todayDate.getDate() + 1));
+    daysList.push({
+      id: i + 1,
+      date: date.toISOString().substring(0, 10),
+      name: date.toLocaleDateString("en-US", { weekday: "short" }),
+    } as DaysList);
+  }
+  return daysList;
+});
+
+const updateDate = (date: string) => {
+  seancesStore.selectedDate = date;
```

Comment:
You could use `storeToRefs` and use `selectedDate` as a reactive variable instead of additional `datePickerDate` as it seems datepickerdate is just mapped to selectedDate anyway

Reply:
Great tip - I can get away with watching a data property!